run_in_docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ $# -lt 3 ]
+then
+    echo "Usage:"
+    echo "./bin/run.sh two_fer ~/input/ ~/output/"
+    exit 1
+fi
+
+Exercise="$1"
+InputDirectory="$2"
+OutputDirectory="$3"
+
+docker build -t exercism/java-test-runner .
+docker run -v ${InputDirectory}:/solution/ -v ${OutputDirectory}:/results/ exercism/java-test-runner $Exercise /solution /results
```

Comment:
I got the build to run in offline mode finally, 😅. However the read-only command doesn't let me make and changes to any of the files within the running container so I am not having any luck on the front. I also didn't see the `--read-only` command on other test runner implementations so was wondering it was optional?
In order for the gradle interceptor to work, I need to add listeners to the solution's `build.gradle` file and those changes should be made within the container and not on the mounted volume dir since it would potentially leave behind side effects. The way it currently works, no changes are made to the host filesystem, all changes are contained in the container.

Reply:
> I got the build to run in offline mode finally

I still have to figure this out for F# :D

> I also didn't see the --read-only command on other test runner implementations so was wondering it was optional?

It isn't. That said, the input and output directory as well as the `/tmp` directory are all writable, so that should be enough for you, right? p.s. the [JavaScript test runner](
https://github.com/exercism/javascript-test-runner/blob/master/run-in-docker.sh#L31) does use read-only in their script. The other test runners should do that too.

> The way it currently works, no changes are made to the host filesystem, all changes are contained in the container.

That should work.